cve_bin_tool/cli.py

```diff
@@ -435,14 +441,22 @@ def main(argv=None):
         version_check = args["disable_version_check"]
         db_update = args["update"]
 
+    output_formats = set(args["format"].split(","))
+    extensions = ["csv", "json", "console", "html", "pdf"]
+    for output_format in output_formats:
+        if output_format not in extensions:
```

Comment:
```suggestion
        if output_format.strip() not in extensions:
```
This would help us solve the space issue (the note in the docs)

Reply:
Reference: https://docs.python.org/3/library/stdtypes.html#str.strip

Reply:
if there is a space between multiple output formats then it will be interpreted as a different argument, right?

Reply:
still this is a good idea because if someone uses it with a quotation mark, then it won't work. thank you